flytekit/bin/entrypoint.py

```diff
@@ -140,17 +143,34 @@ def _execute_task(task_module, task_name, inputs, output_prefix, raw_output_data
                     tmp_dir=user_workspace_dir,
                 )
 
-                with ctx.new_data_proxy_by_cloud_provider(
-                    cloud_provider=cloud_provider, raw_output_data_prefix=raw_output_data_prefix
-                ) as ctx:
+                if cloud_provider == _constants.CloudProvider.AWS:
```

Comment:
how do we get the cloudprovider? Should this not be based on the prefix in the file?

Reply:
It's retrieved here https://github.com/lyft/flytekit/pull/215/files#diff-932d345851356237c198d139fa9a11d53d68257dc05387e933680ed2e764a50fR106